src/test/java/picard/sam/markduplicates/UmiAwareMarkDuplicatesWithMateCigarTester.java

```diff
@@ -228,7 +229,11 @@ public void test() {
         }
 
         // Also do tests from AbstractMarkDuplicatesCommandLineProgramTester
-        super.test();
+        try {
+            super.test();
+        } catch (IOException ex) {
+            Assert.fail("Could not open metrics file: " + ex.getMessage());
```

Comment:
better to include the throwable so you get a stack crawl
```suggestion
            Assert.fail("Could not open metrics file", ex);
```